api/src/main/java/ai/djl/util/cuda/CudaUtils.java

```diff
@@ -155,8 +188,42 @@ public static MemoryUsage getGpuMemory(Device device) {
         return new MemoryUsage(-1, committed, committed, total[0]);
     }
 
+    /**
+     * The main entrypoint to get CUDA information with command line.
+     *
+     * @param args the command line arguments.
+     */
+    @SuppressWarnings("PMD.SystemPrintln")
+    public static void main(String[] args) {
+        int gpuCount = getGpuCount();
+        if (args.length == 0) {
+            if (gpuCount <= 0) {
+                System.out.println("0,-1");
+                return;
+            }
+            int cudaVersion = getCudaVersion();
+            System.out.println(gpuCount + "," + cudaVersion);
```

Comment:
Why use println?

Reply:
We need to read the console output from java side